packages/contentful--app-scripts/lib/utils.js

```diff
@@ -59,9 +62,43 @@ const selectFromList = async (list, message, cachedOptionEnvVar) => {
   }
 };
 
+function getActionsManifest() {
+  const isManifestExists = fs.existsSync(DEFAULT_MANIFEST_PATH);
+
+  if (!isManifestExists) {
```

Comment:
Would it make sense to provide an error here that says "Manifest file required at './contentful-app-manifest.json'" just in case someone changes the name for some reason?

Reply:
But it's not required afaik. So we can't really check if some random file is a manifest or not.

Reply:
Ah right, fair point